static/js/client.js

```diff
@@ -103,14 +103,39 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
     }
   }
 
-  function updateTitle() {
+  //see http://stackoverflow.com/a/9609450
+  var decodeEntities = (function() {
```

Comment:
I was able to remove this when I made direction a plugin, must have been added back with the merge